runtime/ms-rest-azure/lib/subscriptionManagement/models/index.js

```diff
@@ -18,11 +18,8 @@ exports.BaseResource = require('../../baseResource');
 exports.CloudError = require('../../cloudError');
 exports.Location = require('./location');
 exports.LocationListResult = require('./locationListResult');
-exports.SubscriptionPolicies = require('./subscriptionPolicies');
```

Comment:
This one wasn't duplicated, right?

Reply:
I sorted them so it's easier to spot the duplicates in the future.